modules/kubernetes/prometheus/charts/prometheus-extras/values.yaml

```diff
@@ -0,0 +1,27 @@
+# Default values for prometheus-extras.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 2
+
+serviceAccount:
+  name: prometheus
+
+service:
+  type: ClusterIP
+  port: 9090
+
+resources: {}
+
+remoteWrite:
+  enabled: true
+  url: ""
+  name: xenitInfra
```

Comment:
should this really be the default? maybe use something more generic than xenit?

Reply:
Any suggestions? 
Just infra, thanos?

Reply:
+1 for `thanos`